plugins/network-elements/nuage-vsp/src/com/cloud/network/resource/NuageVspResource.java

```diff
@@ -424,9 +449,10 @@ private Answer executeRequest(ApplyAclRuleVspCommand cmd) {
     private Answer executeRequest(ShutDownVpcVspCommand cmd) {
         try {
             isNuageVspElementLoaded();
-            _nuageVspElementClient.shutDownVpc(cmd.getDomainUuid(), cmd.getVpcUuid());
+            _nuageVspElementClient.shutdownVpc(cmd.getDomainUuid(), cmd.getVpcUuid(), cmd.getDomainTemplateName());
             return new ShutDownVpcVspAnswer(cmd, true, "Shutdown VPC " + cmd.getVpcUuid());
         } catch (Exception e) {
```

Comment:
Why are all unchecked and checked exceptions being caught?  It seems like we should only be catching the expected checked exceptions, and allow unchecked exceptions simply bubble out.